portfolio/src/main/webapp/script.js

```diff
@@ -12,18 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+//TODO: Add logout button
 function getComments() {
   fetch('/data').then(response => response.json()).then((json) => {
-        
     const commentsElement = document.getElementById('comments');
     commentsElement.innerText = '';
+    const loggedInStatus = json.loggedInStatus;
```

Comment:
Looks like this is not used. Is this for a follow up PR?

Reply:
Yes, this will be used in a follow up.